appcli/orchestrators.py

```diff
@@ -219,6 +241,19 @@ def task(
         command.extend(list(extra_args))
         return self.__compose_task(cli_context, command)
 
+    def exec(
+        self,
+        cli_context: CliContext,
+        service_name: str,
+        command_name: str,
+        extra_args: Iterable[str] = [],
+    ) -> CompletedProcess:
+        command = ["docker", "exec"]  # Command is: exec SERVICE COMMAND [ARGS]
+        command.append(service_name)
```

Comment:
Without being able to test this myself right now, I'm not sure if this will work. From my understanding, the name of the service (e.g. `myservice`) will _not_ match the name of the actual running container when it is launched with `docker-compose`.

You should probably be using the `self.__compose_service` function, which will call docker-compose with the correct args for you

Reply:
I initially tried using self.__compose_task to run them, but once it got to execute_compose, it would prepend arguments to the command which would cause some commands to have errors, and others to break.
That's why I used __exec_command as a way to run it.